crates/egui/src/containers/window.rs

```diff
@@ -278,9 +292,17 @@ impl<'open> Window<'open> {
     }
 
     /// Set initial collapsed state of the window
+    #[deprecated = "Use `default_collapsed` to set default collapsed state instead. This function might change meaning in future, but for now does the same thing as it used to"]
```

Comment:
The old function was named `default_open` which didn't really align with the usage of `open` to mean whether the window was shown or not. So, I changed to explicitly use `collapsed` for indicating the collapsed state of the window